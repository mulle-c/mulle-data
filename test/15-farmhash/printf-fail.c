#include <stdio.h>


int  main( void)
{
//  printf( "%#016llx\n", 0x00000000ced51853LL);
//  printf( "%#016llx\n", 0x10000000ced51853LL);
//  printf( "%#016llx\n", 0x20000000ced51853LL);

   printf( "%#08x\n", 0x00000000);
   printf( "%#08x\n", 0x00000001);
   printf( "%#08x\n", 0x10000002);
   printf( "%#08x\n", 0x20000003);

   printf( "\n");

   printf( "%#x\n", 0x00000000);
   printf( "%#x\n", 0x00000001);
   printf( "%#x\n", 0x10000002);
   printf( "%#x\n", 0x20000003);

   printf( "\n");

   printf( "%x\n", 0x00000000);
   printf( "%x\n", 0x00000001);
   printf( "%x\n", 0x10000002);
   printf( "%x\n", 0x20000003);
   return( 0);
}