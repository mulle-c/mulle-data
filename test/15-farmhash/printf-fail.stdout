00000000
0x000001
0x10000002
0x20000003

0
0x1
0x10000002
0x20000003

0
1
10000002
20000003
